app/server/services/sendgrid_email.js

```diff
@@ -0,0 +1,117 @@
+var moment = require('moment');
+var ROOT_URL = process.env.ROOT_URL;
+
+var SENDGRID_API_KEY = process.env.SENDGRID_API_KEY;
+var HACKATHON_NAME = process.env.HACKATHON_NAME;
+var EMAIL_ADDRESS = process.env.EMAIL_ADDRESS;
+var TWITTER_HANDLE = process.env.TWITTER_HANDLE;
+var FACEBOOK_HANDLE = process.env.FACEBOOK_HANDLE;
+var ACCEPTANCE_EMAIL_TEMPLATE = process.env.ACCEPTANCE_EMAIL_TEMPLATE;
+var VERIFICATION_EMAIL_TEMPLATE = process.env.VERIFICATION_EMAIL_TEMPLATE;
+var PASSWORD_RESET_EMAIL_TEMPLATE = process.env.PASSWORD_RESET_EMAIL_TEMPLATE;
+var PASSWORD_CHANGED_EMAIL_TEMPLATE = process.env.PASSWORD_CHANGED_EMAIL_TEMPLATE;
+var WAIVER_EMAIL_TEMPLATE = process.env.WAIVER_EMAIL_TEMPLATE;
+
+var EMAIL_CONTACT = process.env.EMAIL_CONTACT;
+var EMAIL_HEADER_IMAGE = process.env.EMAIL_HEADER_IMAGE;
+if(EMAIL_HEADER_IMAGE.indexOf("https") == -1){
```

Comment:
Make sure the coding style is consistent with the rest of the code (aka leaving a space between `if` and the opening parenthesis, as well as the closing parenthesis and the curly braces, using single quotes instead of double quotes, etc.) as well as adding a new line above it for readability purposes.

Is there a reason why we're not supporting plain old HTTP? This statement would evaluate to true if we're passing in a non-HTTPS url and would end up with a broken link.

Another thing is that this statement would evaluate to false if I'm passing in a relative URL that contains the characters `https` in a row. To fix this you can check if the index is 0 instead of -1, as that would ensure that `https` is indeed in the beginning of the url. Adding the `://` is important as well to ensure that it is indeed an absolute url.

```suggestion

if (EMAIL_HEADER_IMAGE.indexOf('http://') == 0 || EMAIL_HEADER_IMAGE.indexOf('https://') == 0) {
```

Or if you want to make it fancier, you can use [regex](https://stackoverflow.com/questions/10687099/how-to-test-if-a-url-string-is-absolute-or-relative) :)

Reply:
Oh, actually this logic isn't even needed at all, since the templates will supposedly all be made via SendGrid portal.

Reply:
Oh, actually this logic isn't even needed at all, since the templates will supposedly all be made via SendGrid portal.

Reply:
Oh, actually this logic isn't even needed at all, since the templates will supposedly all be made via SendGrid portal.

Reply:
Oh, actually this logic isn't even needed at all, since the templates will supposedly all be made via SendGrid portal.